python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1335,6 +1332,26 @@ def test_delta_binary(nrows, add_nulls, tmpdir):
     pcdf = cudf.from_pandas(test_pdf)
     assert_eq(cdf, pcdf)
 
+    # Write back out with cudf and make sure pyarrow can read it
+    cudf_fname = tmpdir.join("cudfv2.parquet")
+    pcdf.to_parquet(
+        cudf_fname,
+        compression=None,
+        header_version="2.0",
+        use_dictionary=False,
+    )
+
+    # FIXME(ets): should probably not use more bits than the data type
+    try:
+        cdf2 = cudf.from_pandas(pd.read_parquet(cudf_fname))
+    except OSError:
+        if dtype == "int32" and nrows == 100000:
+            pytest.mark.xfail(
+                reason="arrow does not support 33-bit delta encoding"
```

Comment:
```suggestion
                reason="arrow does not support 32-bit delta encoding"
```
typo?

Reply:
No, that's actually the correct wording. The cudf delta encoder will use 33 bits when encoding 32-bit integers.  Some parquet readers are ok with this (cudf, parquet-mr), some are not (arrow-cpp, arrow-rs). I have a follow-on to this PR that will restrict the encoding to 32 bits to make everyone happy, then the xfail will go away.

Reply:
TIL. Thanks @etseidl !